### ISSUE TITLE
gtkdialog failing

### ISSUE THREAD

**Issue by deulerb** [amiaopensource/homebrew-amiaos#321]
Hi All, I am trying to install vrecord and am receiving the following errors which i don't know how to fix. Any help would be massively appreciated - thanks in advance. David

Last 15 lines from /Users/seb/Library/Logs/Homebrew/gtkdialog/02.make:
make[3]: *** [printing.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from variables.c:23:
./gtkdialog.h:30:10: fatal error: 'gtk/gtk.h' file not found
#include <gtk/gtk.h>
         ^~~~~~~~~~~
1 error generated.
make[3]: *** [gtkdialog.o] Error 1
1 error generated.
make[3]: *** [gtkdialog_parser.o] Error 1
1 error generated.
make[3]: *** [variables.o] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

**Comment by privatezero** [comment]
Hi @deulerb - thanks for reporting this! We've been able to confirm that there are issues with the vrecord dependency `gtkdialog` that are messing up the overall install on Macs. We'll try to get to the bottom of it to figure out a fix.

**Comment by retokromer** [comment]
@privatezero There are new releases of `gtkdialog` and [0.8.5](https://github.com/puppylinux-woof-CE/gtkdialog/releases/tag/0.8.5) has just been released.

**Comment by deulerb** [comment]
This is amazing news. Thanks for the note. > @privatezero There are new releases of `gtkdialog` and [0.8.5](https://github.com/puppylinux-woof-CE/gtkdialog/releases/tag/0.8.5) has just been released.

**Comment by retokromer** [comment]
@privatezero FYI: With the new 0.8.5 I got the same error
```
gtkdialog.c:37:10: fatal error: 'gtk/gtk.h' file not found
#include <gtk/gtk.h>
         ^~~~~~~~~~~
```
on macOS (both x86_64 and AArch64).

**Comment by deulerb** [comment]
> @privatezero FYI: With the new 0.8.5 I got the same error
> 
> ```
> 
> gtkdialog.c:37:10: fatal error: 'gtk/gtk.h' file not found
> 
> #include <gtk/gtk.h>
> 
>          ^~~~~~~~~~~
> 
> ```
> 
> on macOS (both x86_64 and AArch64).

Same error again for me too!

**Comment by privatezero** [comment]
The fix in https://github.com/amiaopensource/homebrew-amiaos/pull/324 appears to resolve this issue - once that is merged please test again and let us know if it is working for you as well!

**Comment by retokromer** [comment]
resolved in #324
